docs/jwst/msaflagopen/description.rst

```diff
@@ -0,0 +1,31 @@
+
+Description
+===========
+
+Overview
+--------
+
+The msaflagopen correction flags pixels that are affected by msa shutters
```

Comment:
Let's be a bit more explicit and say either "... affected by **NIRSpec MSA** shutters ..." or "... flags pixels **in NIRSpec exposures** that are affected by **MSA** shutters ..."

Reply:
OK